users/vieting/experiments/switchboard/hybrid/feat/experiments.py

```diff
@@ -144,9 +144,14 @@ def run_baseline_scf():
     )
 
     nn_args = get_nn_args_baseline(
+        nn_base_args={
+            "conformer_bs7k_scf": dict(
+                returnn_args=dict(batch_size=7000),
+                feature_args={"class": "ScfNetwork", "size_tf": 256 // 2, "stride_tf": 10 // 2}
+                )
+            },
```

Comment:
Please check the indentation. I meant to include this in the previous review, but apparently my comment wasn't saved.